Packs/Salesforce/IncidentFields/incidentfield-Salesforce_Status.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
@bakatzir @silviudm Consider using "Source Status" instead.

Reply:
fixed

Reply:
fixed

Reply:
@silviudm I don't see that you changed it to "Source Status". Do you thing there's a reason for creating a new incident field here instead of using an existing one?